Mail2Bug/WorkItemManagement/SimpleWorkItem.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections.Generic;
+
+namespace Mail2Bug.WorkItemManagement
+{
+    public class SimpleWorkItem
+    {
+        /// <summary>
+        /// The work item ID.
+        /// </summary>
+        public int Id { get; set; }
+
+        /// <summary>
+        /// Work item title.
+        /// </summary>
+        public string Title { get; set; }

```

Comment:
Other than ID and Title, I don't think any of the others need to be first-class members of this class (even for ID and title, that's debatable) - reason is that if we were to implement support for a different work-item management system in the future, it would not necessarily have these fields.

Instead, they should all fall back to relying on getting the value by field name